gapic/src/main/com/google/gapid/perfetto/views/TraceConfigDialog.java

```diff
@@ -167,6 +171,10 @@ public static String getConfigSummary(Settings settings, Device.PerfettoCapabili
       }
       if (settings.perfettoCpuSlices) {
         ftrace.addAllAtraceCategories(Arrays.asList(CPU_SLICES_ATRACE));
```

Comment:
Can we only add the atrace stuff for Android devices?
We get a warning if you have them enabled on desktop, that would be nice to avoid.

Reply:
Yes that is a good point. I didn't know there was a warning and thought it would just be ignored. There is another issue, which will require the device info to communicate some atrace support information. Can I do it when fixing that?

Reply:
What's the plan here? We found that this change breaks ggp system profiling; see https://github.com/google/gapid/issues/3529.